### PR TITLE
Embedded xBlock sharing

### PR DESCRIPTION
**Pull request by wowkalucky**
@kabirkhan @sdolenc

## Added
- transcripts functionality extracted to separate amp-plugin - transcriptsAmpPlugin;
- embedded player sharing feature;
- assets downloading modes;
(please, refer to [README's corresponding section](https://github.com/raccoongang/xblock-azure-media-services/blob/oxa/embed_player/README.md#assets-downloading));

## Fixed
- xBlock's footer buttons styling;
- xBlock's dependencies updated;

### CODE REVIEW

**Comment by sdolenc** [comment]
we changed the base branch to copy of oxa/dev.fic for now

File: azure_media_services/ams.py [hunk 1]
```diff
@@ -237,6 +285,17 @@ def get_icon_class(self):
                 new_class = higher_class
         return new_class
 
+    def get_embed_url(self):
+        try:
+            embed_url = reverse(
+                'embed_player',
+                kwargs={'usage_key_string': unicode(self.scope_ids.usage_id).encode('utf-8')}
```
Comment (satyarapelly) [inline azure_media_services/ams.py:294]: This required to add the url routing in lms/urls.py and or cms/urls.py
url(
        r'^embed_player/',
        include('azure_media_services.urls')
    ),

**Review by satyarapelly — COMMENTED** [review]
I'm currently applying the PR on oxa/dev.fic. We will approve this later today.

**Comment by satyarapelly** [comment]
Please do not merge, I'm still finding the issue with embed_player, I'm not able play the video anonymously.
I will share my environment that I had setup, you can troubleshoot.

I have setup brand news fullstack environment with oxa/dev.fic and applied this PR. everything seems to be find except embed payer with un-authenticated scenario is not working.

**Review by satyarapelly — APPROVED** [review]

**Comment by satyarapelly** [comment]
Issues:
1.	“Download video” in Chrome actually opening new tab and playing
2.	In EDGE, video itself not visible, only audio.

**Review by satyarapelly — APPROVED** [review]

**Pull request merged**